docs/Contribute-to-documentation.md

```diff
@@ -0,0 +1,16 @@
+
+---
+## Contribute to documentation
+Contributing to the Nebula documentation can be a rewarding experience. When you offer feedback, questions, edits, or new content, you help us, the projects you work on, and the larger Nebula community.
```

Comment:
```
When you offer feedback, questions, edits, or new content, you help us, the projects you work on, and the larger Nebula community.
```
This should be divided into two.

_When you offer feedbacks, questions, edits, or new contents, you are helping us._

Besides, what _the projects you work on, and the larger Nebula community._  is saying?

Reply:
Means users' feedback can help us improve our Nebula community. Deleted that sentence and add
We welcome your participation to help make the documentation better!